packages/http-client-python/generator/pygen/codegen/templates/request_builders.py.jinja2

```diff
@@ -1,6 +1,8 @@
 {% import 'operation_tools.jinja2' as op_tools %}
 # coding=utf-8
-{{ code_model.options['license_header'] }}
+{% if code_model.license_header %}
+{{ code_model.license_header }}
```

Comment:
small nit: should we just have `code_model.license_header` be an empty string, then rely on `black` to do some formatting? that way we don't have to keep doing `if code_model.license_header`